### PR TITLE
Tried native Stop and Go on Highlander ICE with no comma pedal: Works!

### PR DESCRIPTION
**Pull request by zeeexsixare**
I edited Highlander interface.py to try out stop and go like on the Hybrid.  To my surprise it works out of the box!  Video of operation recorded here: https://comma.slack.com/files/UBHSVCMSS/FCPBN09LP/vid_20180906_075803.mp4

### CODE REVIEW

File: selfdrive/car/toyota/carcontroller.py [hunk 1]
```diff
@@ -63,7 +63,7 @@ def process_hud_alert(hud_alert, audible_alert):
     sound1 = 1
   elif audible_alert in ['beepSingle', 'chimeSingle', 'chimeDouble']:
     # TODO: find a way to send single chimes
-    sound2 = 1
+    sound2 = 0 #Changed by P Lee to disable disengage chime
```
Comment (rbiasini) [inline selfdrive/car/toyota/carcontroller.py]: @zeeexsixare : can you remove the changes to `carcontroller.py`? Also, can you update the README by lowering the min accel speed to zero?

**Comment by rbiasini** [comment]
Great! Just a couple of comments to solve and we can merge this

**Comment by zeeexsixare** [comment]
Rick, changes requested have been made.  Let me know if it's good to go.  This is my first pull request ever and first contribution to comma and it's been a blast.  I really appreciate your help.

File: selfdrive/car/toyota/carcontroller.py (hunk 1 above)
Comment (zeeexsixare) [reply selfdrive/car/toyota/carcontroller.py]: This is done now.

File: selfdrive/car/toyota/carcontroller.py [hunk 2]
```diff
@@ -9,7 +9,7 @@
 
 # Accel limits
 ACCEL_HYST_GAP = 0.02  # don't change accel command for small oscilalitons within this value
-ACCEL_MAX = 1.5  # 1.5 m/s2
+ACCEL_MAX = 1.5  # 1.5 m/s2
```
Comment (rbiasini) [inline selfdrive/car/toyota/carcontroller.py]: trailing space

File: README.md [hunk 3]
```diff
@@ -107,7 +107,7 @@ Community Maintained Cars are not confirmed by comma.ai to meet our [safety mode
 In Progress Cars
 ------
 - All TSS-P Toyota with Steering Assist.
-  - 'Full Speed Range Dynamic Radar Cruise Control' is required to enable stop-and-go. Only the Prius, Camry and C-HR have this option.
+  - 'Full Speed Range Dynamic Radar Cruise Control' is required to enable stop-and-go. Only the Prius, Camry and C-HR have this option.  OpenPilot also adds stop-an-go functionality for Highlander and Highlander Hybrid.
```
Comment (rbiasini) [inline README.md]: I would remove this change. But yes, later on we'll add something to indicate which cars gain stop and go.

File: selfdrive/car/toyota/carcontroller.py (hunk 2 above)
Comment (zeeexsixare) [reply selfdrive/car/toyota/carcontroller.py]: Fixed

File: README.md (hunk 3 above)
Comment (zeeexsixare) [reply README.md]: Fixed

**Comment by zeeexsixare** [comment]
Also, cabana of the drive where it worked at 7:57AM eastern time (480 seconds into the drive):
https://community.comma.ai/cabana/?route=aa659debdd1a7b54%7C2018-09-06--07-49-35&max=10&url=https%3A%2F%2Fchffrprivate.blob.core.windows.net%2Fchffrprivate3%2Fv2%2Faa659debdd1a7b54%2F69a844a3a0cb3d3dbc7830c82b9673f1_2018-09-06--07-49-35

**Comment by rbiasini** [comment]
nice job finding this out!

**Pull request merged**